lib/fizzbuzz.rb

```diff
@@ -3,6 +3,15 @@ class SuperFizzBuzz
   def run(input)
 
     #Implement your code here
+    if input % 3 == 0 and input % 5 == 0
+      return "FizzBuzz"
+    elsif input % 3 == 0
+      return "Fizz"
+    elsif input % 5 == 0
+      return "Buzz"
+    else
+      return input
+    end
```

Comment:
omit unnecessary uses of `return`.  Ruby methods automatically return the value of the last line evaluated.